packages/cli/src/databases/entities/test-run.ee.ts

```diff
@@ -35,4 +35,17 @@ export class TestRun extends WithTimestampsAndStringId {
 
 	@Column(jsonColumnType, { nullable: true })
 	metrics: AggregatedTestRunMetrics;
+
+	// Total number of the test cases, matching the filter condition of the test definition (specified annotationTag)
+	@Column('integer', { nullable: true })
+	total: number;
```

Comment:
We seem to refer to this as `totalCases` in the code. Maybe all of these could have that suffix in the name that also in the DB?